llvm/test/MC/Disassembler/AMDGPU/gfx11_dasm_vop1_dpp16.txt

```diff
@@ -1765,6 +1765,18 @@
 # GFX11: v_mov_b32_dpp v255, v255 row_xmask:15 row_mask:0x3 bank_mask:0x0 bound_ctrl:1 fi:1 ; encoding: [0xfa,0x02,0xfe,0x7f,0xff,0x6f,0x0d,0x30]
 0xfa,0x02,0xfe,0x7f,0xff,0x6f,0x0d,0x30
 
+# GFX11: v_mov_b32_e64_dpp v5, -v1 quad_perm:[3,2,1,0] row_mask:0xf bank_mask:0xf ; encoding: [0x05,0x00,0x81,0xd5,0xfa,0x00,0x00,0x20,0x01,0x1b,0x00,0xff]
```

Comment:
This, and other new cases below are VOP3. This should be in gfx11_dasm_vop3_dpp16_from_vop1.txt